<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choice</class>
 <widget class="QMainWindow" name="choice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="bg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../images/bg_1.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="room1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>151</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/math_1.png</normaloff>../images/math_1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>301</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="room2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>151</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/chem_1.png</normaloff>../images/chem_1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>301</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="room3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>160</y>
      <width>151</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/prog_1.png</normaloff>../images/prog_1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>301</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="room4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>160</y>
      <width>151</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/dbms_1.png</normaloff>../images/dbms_1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>150</width>
      <height>301</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
